cube.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    contextToAppId: ({ securityContext }) => `CUBEJS_APP_${securityContext.tenantSchema}`,
+    checkAuth: (req, token) => {
+        const tenantSchema = req.headers['organization']
+        req.securityContext = { tenantSchema }
+    },
+    extendContext: async (req) => {
+        return {
+            'organizationTenantSchema': 'abcd',
+        };
+    },
```

Comment:
do we need this?

Reply:
not needed. I missed pushing the latest commit.